ui/tract-renderer/src/main/java/org/cru/godtools/tract/viewmodel/CardViewHolder.java

```diff
@@ -114,6 +131,75 @@ private void bindLabel() {
         mDivider.setBackgroundColor(Styles.getTextColor(mModel));
     }
 
+    private void bindCardNavigation() {
+        int cardPositionCount = mModel != null ? mModel.getPosition() + 1 : 1;
+        Locale locale = mModel != null ? mModel.getManifest().getLocale() : Locale.getDefault();
+        int cardCount = getCardCount();
+        String positionText = String.format(locale, "%d/%d", cardPositionCount, cardCount);
+        mCardPositionView.setText(positionText);
+        if (isPrayerForm(mModel) || isPrayerSelection(mModel)) {
+            mPreviousCardView.setVisibility(View.INVISIBLE);
+            mPreviousCardView.setEnabled(false);
+            mCardPositionView.setVisibility(View.INVISIBLE);
+            mNextCardView.setVisibility(View.INVISIBLE);
+            mNextCardView.setEnabled(false);
+            return;
+        }
+        Context localContext = LocaleUtils.localizeContextIfPossible(mCardView.getContext(), locale);
+        if (cardPositionCount == 1) {
+            mPreviousCardView.setVisibility(View.INVISIBLE);
+            mPreviousCardView.setEnabled(false);
+        } else {
+            mPreviousCardView.setVisibility(View.VISIBLE);
+            mPreviousCardView.setEnabled(true);
+            mPreviousCardView.setText(localContext.getString(R.string.previous));
+        }
+
+        if (cardPositionCount == cardCount) {
+            mNextCardView.setVisibility(View.INVISIBLE);
+            mNextCardView.setEnabled(false);
+        } else {
+            mNextCardView.setVisibility(View.VISIBLE);
+            mNextCardView.setEnabled(true);
+            mNextCardView.setText(localContext.getString(R.string.next));
+        }
+
+    }
+
+    private boolean isPrayerForm(Card card) {
+        if (card != null) {
+            for (Content content : card.getContent()) {
+                if (content instanceof Form) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    private boolean isPrayerSelection(Card card) {
+        if (card != null) {
+            for (Event.Id dismissListener : card.getDismissListeners()) {
+                if (dismissListener.name.contains("followup-form")) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    private int getCardCount() {
+        int count = 0;
+        if (mModel != null) {
+            for (Card card : mModel.getPage().getCards()) {
+                if (!isPrayerForm(card) || !isPrayerSelection(card)) {
```

Comment:
Should this be `&&` instead of `||`? Maybe something clearer would be:
```
if (isPrayerForm(card) || isPrayerSelection(card)) {
  continue;
}
count++;
```